library/src/main/java/com/google/android/exoplayer/hls/HlsPlaylistParser.java

```diff
@@ -152,18 +153,20 @@ private static HlsMasterPlaylist parseMasterPlaylist(LineIterator iterator, Stri
         String type = HlsParserUtil.parseStringAttr(line, TYPE_ATTR_REGEX, TYPE_ATTR);
         if (SUBTITLES_TYPE.equals(type)) {
           // We assume all subtitles belong to the same group.
-          String name = HlsParserUtil.parseStringAttr(line, NAME_ATTR_REGEX, NAME_ATTR);
+          name = HlsParserUtil.parseStringAttr(line, NAME_ATTR_REGEX, NAME_ATTR);
```

Comment:
Yes I will change this tomorrow, the main reason I reused this variable was because it already had the identifier `name`.